lib/Parse/ParseType.cpp

```diff
@@ -650,7 +649,7 @@ ParserStatus Parser::parseGenericArguments(SmallVectorImpl<TypeRepr *> &Args,
     SyntaxParsingContext ListContext(SyntaxContext,
         SyntaxKind::GenericArgumentList);
 
-    while (true) {
+    for (;;) {
```

Comment:
Let's leave these alone; one is not better than the other, and `while (true)` is used commonly in this codebase, so it's best not to cause churn.